ci/citest/acceptance-test/multibox/ind-steps/step-vdc-images/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+for img in "${IMAGES[@]}" ; do
+    item=$(grep "$img" <<< "$imgIndex")
+    IFS=';' read -a parts <<< "$item"
+    IFS=';' eval 'folders=($item)'
+    imgRemoteDir="${parts[-1]}"
+
+    prepare_container_image "${img}" "${folders[0]}/${folders[1]}/${folders[2]}" "$imgRemoteDir"
+    add_container_image "${img}" "${folders[0]}/${folders[1]}/${folders[2]}"
+done
+
+(
+    # resync again after images have been downloaded
+    $starting_step "Synching guestroot for ${vm_name}"
```

Comment:
Sounds like a reasonable change, rsync itself might as well be a stage then everything else can be kept as it is.